app/controllers/superheros_controller.rb

```diff
@@ -7,6 +7,16 @@ def create
         superhero = Superhero.create(hero_params)
         render json: superhero
     end
+    def update
+        superhero = Superhero.find(params[:id])
+        superhero.update(hero_params)
+        if superhero.valid?
+            render json: superhero
+        else
+            render json: superhero.errors
```

Comment:
is there a specific status that should be returned here for when things don't go as planned?